doc/manual/citing_aspect.bib

```diff
@@ -2450,3 +2450,43 @@ @article{Jang2022
   pages = {179–190}
 }
 
+@article{weerdesteijn2024recent,
+title={Recent ice melt above a mantle plume track is accelerating the uplift of Southeast Greenland},
+author={Weerdesteijn, Maaike FM and Conrad, Clinton P},
+journal={Communications Earth \& Environment},
+volume={5},
+number={1},
+pages={791},
+year={2024},
+doi={10.1038/s43247-024-01968-6},
+url={https://doi.org/10.1038/s43247-024-01968-6},
+publisher={Nature Publishing Group UK London}
+}
+
+@article{https://doi.org/10.1029/2023GC011380,
+author = {Heyn, Björn H. and Shephard, Grace E. and Conrad, Clinton P.},
+title = {Prolonged Multi-Phase Magmatism Due To Plume-Lithosphere Interaction as Applied to the High Arctic Large Igneous Province},
+journal = {Geochemistry, Geophysics, Geosystems},
+volume = {25},
+number = {6},
+pages = {e2023GC011380},
+keywords = {large igneous provinces, mantle plumes, melt migration, plume-lithosphere interaction, artic tectonics and volcanism, plume-induced melting},
+doi = {https://doi.org/10.1029/2023GC011380},
+url = {https://agupubs.onlinelibrary.wiley.com/doi/abs/10.1029/2023GC011380},
+eprint = {https://agupubs.onlinelibrary.wiley.com/doi/pdf/10.1029/2023GC011380},
+note = {e2023GC011380 2023GC011380},
+year = {2024}
+}
+
+@article{Wang_Király_Conrad_Hansen_Fraters_2024,
```

Comment:
can you remove the special characters in the key?

Reply:
Done, seems to work.